node/config/types.go

```diff
@@ -309,6 +309,10 @@ type Pubsub struct {
 	DirectPeers           []string
 	IPColocationWhitelist []string
 	RemoteTracer          string
+	JsonTracer            string
+	ElasticSearchTracer   string
+	ElasticSearchIndex    string
+	TracerSourceAuth      string
```

Comment:
Needs to be hooked up to things in node/builder. Also would be great to add docstrings for those, as we then use those to generate doc comments in the config file